Support for Mac M1.

It looks like Electron now supports the M1 darwin-arm64 architecture, but this package hasn't upgraded to support arm64:

yarn add jest-electron:
```
...
node_modules/electron/install.js:53
  throw err
  ^

Error: Failed to find Electron v6.1.12 for darwin-arm64 at https://github.com/electron/electron/releases/download/v6.1.12/electron-v6.1.12-darwin-arm64.zip
    at Request.<anonymous> (/Users/robblovell/code/node-monorepo/node_modules/nugget/index.js:169:61)
    at Request.emit (node:events:378:20)
    at Request.onRequestResponse (/Users/robblovell/code/node-monorepo/node_modules/request/request.js:1059:10)
    at ClientRequest.emit (node:events:378:20)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (node:_http_client:636:27)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:129:17)
    at TLSSocket.socketOnData (node:_http_client:502:22)
    at TLSSocket.emit (node:events:378:20)
    at addChunk (node:internal/streams/readable:313:12)
    at readableAddChunk (node:internal/streams/readable:288:9)
```
npm add jest-electron:
```
...
npm ERR! code 1
npm ERR! path /Users/robblovell/code/test/node_modules/electron
npm ERR! command failed
npm ERR! command sh -c node install.js
Downloading tmp-72942-0-electron-v6.1.12-darwin-arm64.zip
npm ERR! Error: GET https://github.com/electron/electron/releases/download/v6.1.12/electron-v6.1.12-darwin-arm64.zip returned 404
npm ERR! /Users/robblovell/code/test/node_modules/electron/install.js:53
npm ERR!   throw err
npm ERR!   ^
npm ERR! 
npm ERR! Error: Failed to find Electron v6.1.12 for darwin-arm64 at https://github.com/electron/electron/releases/download/v6.1.12/electron-v6.1.12-darwin-arm64.zip
npm ERR!     at Request.<anonymous> (/Users/robblovell/code/test/node_modules/nugget/index.js:169:61)
npm ERR!     at Request.emit (node:events:378:20)
npm ERR!     at Request.onRequestResponse (/Users/robblovell/code/test/node_modules/request/request.js:1059:10)
npm ERR!     at ClientRequest.emit (node:events:378:20)
npm ERR!     at HTTPParser.parserOnIncomingClient [as onIncoming] (node:_http_client:636:27)
npm ERR!     at HTTPParser.parserOnHeadersComplete (node:_http_common:129:17)
npm ERR!     at TLSSocket.socketOnData (node:_http_client:502:22)
npm ERR!     at TLSSocket.emit (node:events:378:20)
npm ERR!     at addChunk (node:internal/streams/readable:313:12)
npm ERR!     at readableAddChunk (node:internal/streams/readable:288:9)

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/robblovell/.npm/_logs/2021-03-26T19_12_17_979Z-debug.log
```
Node 15 is required for an arm64 architecture machine.

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     ]
   },
   "dependencies": {
-    "electron": "^6.0.7",
+    "electron": "^11.4.6",
     "jest-haste-map": "~24.9.0",
     "jest-message-util": "~24.9.0",
     "jest-mock": "~24.9.0",

